examples/205-automl.ipynb

```diff
@@ -25,6 +25,7 @@
     "- [General AutoML](#chapter_2)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.            model=LinearPerSegmentModel(fit_intercept=True),
</pre><p>Do we need this <code>fit_intercept=True</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/504/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>